packages/core/src/checkout-buttons/strategies/braintree/braintree-venmo-button-strategy.ts

```diff
@@ -172,13 +198,17 @@ export default class BraintreeVenmoButtonStrategy implements CheckoutButtonStrat
             }
         });
 
-        venmoButton.addEventListener('mouseenter', () => {
-            venmoButton.style.backgroundColor = venmoButtonStyleHover.backgroundColor;
-        });
+        if (color === 'blue') {
```

Comment:
Do we need to cover hover state for "white" button? 

It might be better to check how it works in PP Venmo button doc

Reply:
Yes, because if button is white we have to use blue 'Venmo' icon inside it.

Reply:
Can we use `PaypalButtonStyleColorOption.Blue` here?

Reply:
Updated